src/components/home/Home.tsx

```diff
@@ -1,54 +1,53 @@
 import "./Home.css";
 import teas from "../../images/teas.svg";
-// import spilledTea from "../../Coffee-Burst.svg";
 import { NavLink, useNavigate } from "react-router-dom";
-import { useEffect  } from 'react';
+import { useEffect } from "react";
+import { allHardcodedTeas } from '../../utils/interface';
 
 interface Props {
   isRedirected: boolean | any;
-  setIsRedirected: React.Dispatch<React.SetStateAction<boolean | undefined>>
+  setIsRedirected: React.Dispatch<React.SetStateAction<boolean | undefined>>;
 }
 
 function Home({ isRedirected, setIsRedirected }: Props) {
-  const navigate = useNavigate()
+  const navigate = useNavigate();
   useEffect(() => {
     // window.onload = () => {
-      //    setIsRedirected(false)
+    //    setIsRedirected(false)
     //      navigate('/', {replace: true})
     //  }
-      setIsRedirected(false)
-
-}, [navigate, setIsRedirected])
+    setIsRedirected(false);
+  }, [navigate, setIsRedirected]);
 
   return (
     <div>
       <div className="intro-wrapper">
         <article className="article-intro">
-          <img src={teas} />
+          <img src={teas} alt='Cupcake baking pan with different tea blends filling each cup' />
           <div className="info">
-            <h2 className="home-h2">Home</h2>
-            <p>
-              Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do
-              eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim
-              ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut
-              aliquip ex ea commodo consequat.
+            <h2 className="home-h2">Let's 'spill the tea' on tea!</h2>
+            <p className="info-desc">
+              Did you know that all 'teas' originate from the leaves of the Camellia
+              Sinensis plant?(break this up) Any 'tea' that is not from this plant is actually an herbal blend! (break here) What makes each tea unique is
+              the creation process is goes through, including:
```

Comment:
This is fabulously engaging content! Thank you for working on getting the intros together @AsherSpurr !